Gemfile.lock

```diff
@@ -381,6 +381,8 @@ GEM
     nio4r (2.7.3)
     nokogiri (1.16.7-arm64-darwin)
       racc (~> 1.4)
+    nokogiri (1.16.7-x86_64-darwin)
```

Comment:
This file should not have been modified since you are not updating gemfile. However, it seems is something related to your hardware (because uf the platform update).

I feel fine leaving this change, just saying :)

Reply:
yeah, exactly. it’s related to the my Mac's processor, it was added by the `bundle install`